ops/docker-compose.yml

```diff
@@ -111,3 +111,15 @@ services:
         L2_NODE_WEB3_URL: http://l2geth:8545
         URL: http://deployer:8081/addresses.json
         SEQUENCER_PRIVATE_KEY: "0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d"
+
+  integration_tests:
+    image: ethereumoptimism/integration-tests
+    scale: 0
```

Comment:
Using `scale` with a v3 docker compose file appears to break on some versions of docker: https://github.com/ethereum-optimism/optimism/pull/692

It works on my machine and in CI